packages/app-store/server.ts

```diff
@@ -56,7 +57,7 @@ export async function getLocationGroupedOptions(
       ...idToSearchObject,
       app: {
         categories: {
-          hasSome: [AppCategories.conferencing, AppCategories.video],
+          hasSome: defaultVideoAppCategories,
```

Comment:
Reused `defaultVideoAppCategories` here so that we can change it at one place and it would automatically handle querying and client side filtering.